src/qt/sendcoinsdialog.cpp

```diff
@@ -403,7 +403,7 @@ void SendCoinsDialog::presentPSBT(PartiallySignedTransaction& psbtx)
     CDataStream ssTx(SER_NETWORK, PROTOCOL_VERSION);
     ssTx << psbtx;
     GUIUtil::setClipboard(EncodeBase64(ssTx.str()).c_str());
-    QMessageBox msgBox;
+    QMessageBox msgBox(this);
```

Comment:
nit: Is there any specific reason for this change? Without it, everything seems to work still fine on my side (but obviously, it shouldn't hurt to construct the message box with a parent...)

Reply:
It was because I was accessing the parent to click on the message box from the unit test (the dialog `exec()` method blocks the thread until the user presses any of the buttons). And, later on, changed my mind and accessed it via the `QApplication::topLevelWidgets` call.

It doesn't hurt to have it. This would had been a leak if the message box would had been constructed via a `new` call.